app/uk/gov/hmrc/customerprofile/config/microserviceWiring.scala

```diff
@@ -1,50 +0,0 @@
-/*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package uk.gov.hmrc.customerprofile.config
-
-import org.apache.pekko.actor.ActorSystem
-import com.google.inject.Inject
-import com.typesafe.config.Config
-import play.api.Configuration
-import play.api.libs.ws.WSClient
-import uk.gov.hmrc.http.HttpClient
-import uk.gov.hmrc.http.hooks.{HttpHook, HttpHooks}
-import uk.gov.hmrc.play.audit.http.HttpAuditing
-import uk.gov.hmrc.play.audit.http.connector.AuditConnector
-import uk.gov.hmrc.play.http.ws._
-
-import javax.inject.Named
-
-trait Hooks extends HttpHooks with HttpAuditing {
```

Comment:
Why was this file removed?

Reply:
Not Needed anymore after HttpClientV2 injection